lib/DescriptionFileUtils.js

```diff
@@ -1,31 +1,35 @@
+"use strict";
+
 /*
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-var forEachBail = require("./forEachBail");
 
-function loadDescriptionFile(resolver, directory, filenames, callback) {
+const assign = require("object-assign"); //eslint-disable-line
+const forEachBail = require("./forEachBail");
+
+const loadDescriptionFile = (resolver, directory, filenames, callback) => {
 	(function findDescriptionFile() {
 		forEachBail(filenames, function(filename, callback) {
 			var descriptionFilePath = resolver.join(directory, filename);
 			if(resolver.fileSystem.readJson) {
-				resolver.fileSystem.readJson(descriptionFilePath, function(err, content) {
+				resolver.fileSystem.readJson(descriptionFilePath, ((err, content) => {
```

Comment:
I may have to refactor more if I remove the brackets. I get an `unexpected token if` `parsing error`